Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -554,6 +578,69 @@ def setup_(self, step, stepName, stepDict, k, properties):
     offset = 0.522,
 )
 
+upgradeWFs['PatatrackCPU'] = PatatrackWorkflow(
+    digi = {
+        # there is no customisation for enabling the Patatrack pixel quadruplets running only on the CPU
+    },
+    reco = {
+        # skip the @pixelTrackingOnlyValidation which cannot run together with the full reconstruction
```

Comment:
see https://hypernews.cern.ch/HyperNews/CMS/get/recoTracking/1939.html

Reply:
@sroychow @arossi83 @tsusa @connorpa please take note of this issue triggered in the phase1 pixel validation code when using the pixelonly reconstruction (by the way if the issue is only in `SiPixelPhase1HitsV` I suppose one could cook temporarily a lighter ad hoc validation sequence, just removing that module).